---
layout: post
title:  "Next-Generation Pathology"
date:   2020-02-01 12:02:01
categories: Biology
author: Aly O. Abdelkareem
---

![Graphical Abstract](https://i.ibb.co/tPr88dB/Picture1.png){:class="img-responsive"}

**A new stratification of breast cancer patients is based on spatial and cellular resolution. The question is, could imaging mass cytometry (21st-century microscopy) provide sufficiently deep resolution data for re-searchers to understand disease outcomes? And could it support the transition to personalized medicine?**


Current single-cell phenotyping methods can characterize and quantify protein expression on large numbers of live cells with many parameters and sensitivities. Immuno-histochemistry (IHC) is used to study diseased tissue by detecting surface proteins using antibodies, but the method has many limitations. Specifically, it is hard to increase the parameters you are interested in because high-dimensional panels are hard to design. Mass cytometry allows for the simultaneous detection markers with metal iso-topes as barcoding, but cells are destroyed in the process. Mass cytometry is considered an excellent way to assess complex tissues with heterogeneous cell populations since it answers fundamental questions about the different cells--their looks, their types, how they affect each other and function, and their roles in a certain context such as the development/disease. By increasing the size of the panel size and the set of markers, you can ask about the entire system, rather than specific cell types.

The problem is that when we use conventional mass cytometry, we lose the spatial information about cell-to-cell interactions within tissue due to the cells being in suspension. We can only measure whether structural information by imaging, or single-cell information by mass cytometry. In many situations, you must have spatial information in order to answer biological questions. For example, the immune system works by immune cells physically touching and interacting with cancer cells. Thus, we would like to see this interaction, or the distribution of immune cells and to understand the nature of these cells. Using Image Mass Cytometry (IMC), coupled with computational techniques, we can now measure spatial and pheno-type features parallel to each other and to understand its relationship. It also enables us to understand the biology and complexity of tumours. However, there is a trade-off between mass cytometry and IMC (the higher the parameters, the lower the throughput).  Being informed about which molecule is the most important to predict therapy response, we can also see that therapeutic approaches most likely to benefit the patients. It is a break-through as impressively the electron microscopy itself. 

For a decade, Bodenmiller and colleagues were developing experimental and computational methods to study tumour tissue at the single-cell level.  Six years ago, Giesen et al. 1 combined immunohistochemical and immunocytochemical meth-ods and they introduced a new, high-resolution IMC that can measure up to 100 markers at a cellular resolution of 1 µm. Then they applied this technique on breast tumours samples.  Breast tumours are stratified clinically, based on a molecular basis into luminal A/B, basal, and triple-negative 2. Using classical pathological tumour grading meth-ods. we can predict patient prognosis and assess the morphological deviation of tissue and cells. However, patients’ responses to therapy change within clinical subtypes. So a better stratification scheme should be proposed. In clinical subtypes of breast tumours, common signatures and intratumoral heterogeneity were discovered in carcinoma cells using single-cell RNA and DNA sequencing 3,4. If we know the cell heterogeneity and the characteristics of the tumour microenvironment, then, we may discover better means of prognosis and treatment. We will also discover immune cells that can infiltrate tumours, and their crucial, cancer-related pathways.  

Bodenmiller and colleagues ana-lyzed 21 breast cancer samples to validate their IMC technique. Their findings revealed several subgroups within the clinical subtypes defined by traditional histology. They also obtained a detailed view of the spatial information by expression profiling, noting that the tumours near the stromal cells showed the micro-environment effect on those tumour cells. IMC succeeded in overcoming some of the limitations of IHC such as results reproducibility by as-sessing the same protein with multiple antibodies. As a follow-up study, Bodenmiller (Wagner et al. 5) analyzed millions of cells from 144 breast tumours, using mass cytometry. They discovered various pheno-typic abnormalities and tumour-immune cell relationships in breast cancer ecosystems. These were linked to poor prognosis and immunosuppression in both high-grade ER+/- although they were not typically associated with resistance to therapy. These results clarified the failure of clinical trials on check-point inhabitation therapy in breast cancer, unlike in melanoma.

Recently, Bodenmiller (Jackson et al. 6) lab-analyzed breast tumour tissue from 352 patients. All clinical subtypes and grades in the molecular and histology classification 5, were represented.  More than 800K cells were identified and then segmented into a tumour and stromal regions besides the quantification of marker genes and spatial features of each cell. Phonograph 7 (unsupervised clustering) could cluster these cells into 27 cellular metaclusters. These represented the immune, stromal, and epithelial cells, as well as diverse tumour cell phenotypes. It showed immune cells being separated from the tumour mass. In some samples, however, rare cell populations with a low expression of hormone receptors (HR) cells without cytokeratins had invaded the tumour-stroma. Additionally, these cell clusters showed different levels of cytokeratins, HER2 and HR across all metaclusters, correspond-ing to existing clinical subtypes. 

The authors went on to show that these metaclusters never correlated by applying neighbourhood analysis. Each tumour population either had interaction between similar cells or a few interactions with heterogeneous collections. The latter included blood vessels, epithelial areas, stroma areas (immune cells interact-ed), fibroblasts, enclosed endothelial cells in large blood vessels, T cells and proliferating cells surrounding endothelial cells. This technique could also predict, better than clinical data, which cell sub-population best discriminated among patient groups, and which cells differed from the samples taken from the primary diagnosis and after relapse 8. The results revealed that the cell interaction signatures for a stromal-tumour interface cor-related to the grade of the tumour which was scored by a pathologist using the conventional techniques of histology.  

Notably, the authors introduced new 18 subgroups for the stratification of breast tumours based on single-cell spatial and phenotypic features which split the classical subtypes assessed by single stains. Further-more, this stratification was supported by the evidence: multiple cell populations were observed in every subtype. This indicated the failure of classical pathology in tissue classification. The new stratification scheme was more highly correlated with clinical outcomes than the existing classification. For example, with Luminal A tumours, which typically have good survival rates, the patients didn’t survive if the tumours had a CK7 expression. In addition, TNBC had poor outcomes in a subgroup that lacked luminal epithelial markers with high levels of other markers. This was because of its high heterogeneity.  However, another subgroup that split TNBC had a good prognosis if the tumour expressed apoptotic markers and P53. Interestingly, two clusters ex-pressed HR highly and had the same metaclusters, but they differed in prognosis and structure because the small community had lower expression of CK and HR. 

Compared with classical clinical grading, could IMC capture different patients’ outcomes? A new layer of the cellular spatial data was added, patients with spatially distinct communities and heterogeneity of pheno-types had poorer outcomes. These outcomes were associated with spatially defined cell communities and not phenotypes or cellular metaclusters. The authors next showed an association of the stromal environment with specific tumour cell phenotypes across new subgroups. This highlighted the reduced importance of stroma cell phenotypes in identifying clinical subtypes, as they were less informative. One of the most important results was the new-found features associated with survival rates. These were obtained by ana-lyzing the epithelial and stromal communities. The authors observed a correlation between survival outcomes and the community of tumour cells for some cell types. For example, a better outcome was as-sociated with high inflammation, even though this was more common in high-risk TNBC tumours than other clinical subgroups.

One of the main advantages of the Bodenmiller study is that clinical data is available concerning patients’ samples and clinical data. Furthermore, incorporating IMC with other imaging techniques in the clinical management data could provide more information and hopefully, discover more biomarkers. The reproducibility of this method to other cancer tissues might be a concern, but the study was strongly validated when they analyzed another cohort containing 400K cells with the same analytical approach in addition to observing all the cellular metaclusters and subgroups similar to the main cohort. With this deep resolution, we could analyze the complexity of the term ecosystem on a large cohort of patients.  

Bodenmiller's and colleagues’ results raise the possibility of using a single-cell pathology method (IMC) to in-form prognosis beyond the existing clinical classifications. Moreover, it would help us to understand which features of the human ecosystem were predictive of the progression and outcome of the disease. The hope was that some of these sub-populations and the signalling state would correlate with the clinical data and improve patient diagnosis. The long-term impact of these findings would not be limited to the land-scape of breast tumours tissues. Rather, they would lead to new studies on various types of tumours. 

## References

1.	Giesen, C. et al. Highly multiplexed imaging of tumor tissues with subcellular resolution by mass cytometry. Nat. Methods 11, 417–422 (2014).
2.	Malhotra, G. K., Zhao, X., Band, H. & Band, V. Histological, molecular and functional subtypes of breast cancers. Cancer Biology and Therapy vol. 10 955–960 (2010).
3.	Chung, W. et al. Single-cell RNA-seq enables comprehensive tumour and immune cell profiling in primary breast cancer. Nat. Commun. 8, (2017).
4.	Nik-Zainal, S. et al. The life history of 21 breast cancers. Cell 149, 994–1007 (2012).
5.	Wagner, J. et al. A Single-Cell Atlas of the Tumor and Immune Ecosystem of Human Breast Cancer. Cell 177, 1330-1345.e18 (2019).
6.	Jackson, H. W. et al. The single-cell pathology landscape of breast cancer. Nature (2020) doi:10.1038/s41586-019-1876-x.
7.	Levine, J. H. et al. Data-Driven Phenotypic Dissection of AML Reveals Progenitor-like Cells that Correlate with Prognosis. Cell (2015) doi:10.1016/j.cell.2015.05.047.
8.	Bombonati, A. & Sgroi, D. C. The molecular pathology of breast cancer progression. Journal of Pathology vol. 223 307–317 (2011).

